### PR TITLE
RUN-16337 remove unsupported flag

### PR DESCRIPTION
**Pull request by lavianalon**

### CODE REVIEW

**Comment by lavianalon** [comment]
@itayvallach FYI

**Review by itayvallach — APPROVED** [review]

File: docs/Researcher/cli-reference/new-cli/runai.md [hunk 1]
```diff
@@ -17,8 +17,9 @@ runai [flags]
 ## Installing the Improved Command Line Interface
 
 !!! Note
-    * Make sure the flag for the enhanced command line interface is enabled. To enable the Improved Command Line Interface Press the *Tools and Settings* icon, then *General*, then *Workloads*, then enable the *Improved Command Line Interface* toggle.
-    * Only clusters that are version 2.18 or later are supported.
+* Make sure the flag for the enhanced command line interface is enabled. To enable the Improved Command Line Interface Press the *Tools and Settings* icon, then *General*, then *Workloads*, then enable the *Improved Command Line Interface* toggle.
+* Only clusters that are version 2.18 or later are supported.
+* Only Linux and Mac OS are currently supported
```
Comment (jasonnovichRunAI) [inline docs/Researcher/cli-reference/new-cli/runai.md:23]:
```suggestion
* Make sure the flag for the Improved command line interface is enabled. To enable the Improved Command Line Interface Press the *Tools and Settings* icon, then *General*, then *Workloads*, then enable the *Improved Command Line Interface* toggle.

* Only clusters that are version 2.18 or later are supported.

* Only Linux and Mac OS are currently supported

```

Comment (jasonnovichRunAI) [reply docs/Researcher/cli-reference/new-cli/runai.md:23]: @lavianalon please see the correction above.

File: docs/Researcher/cli-reference/new-cli/runai.md [hunk 2]
```diff
@@ -28,36 +29,9 @@ To install the Improved Command Line Interface:
...
-```bash
-[[ -r “/usr/local/etc/profile.d/bash_completion.sh” ]] && . “/usr/local/etc/profile.d/bash_completion.sh”
-
-source <(runai completion bash)
-```
-
 ### Authenticating your CLI
 
-After you have configured your shell, you will need to login to authenticate the CLI.
+First you will need to login to authenticate the CLI.
```
Comment (jasonnovichRunAI) [inline docs/Researcher/cli-reference/new-cli/runai.md:34]:
```suggestion
You will need to login to authenticate the CLI.
```
@lavianalon there is no second, so "first" is not needed.

**Pull request merged**